src/libs/ajax.js

```diff
@@ -874,7 +874,7 @@ const Workspaces = signal => ({
 
       listImportJobs: async isRunning => {
         // ToDo: This endpoint should be deprecated in favor of more generic "importJob" endpoint
-        const res = await fetchOrchestration(`api/${root}/importPFB?running_only=${isRunning}`, _.merge(authOpts(), { signal }))
+        const res = await fetchOrchestration(`api/${root}/importJob?running_only=${isRunning}`, _.merge(authOpts(), { signal }))
         return res.json()
```

Comment:
If you'd like the concision, we've also been using:
```suggestion
        return fetchOrchestration(`api/${root}/importJob?running_only=${isRunning}`, _.merge(authOpts(), { signal })).then(res => res.json())
```

Reply:
We've been doing it in the two line format traditionally; I'd **really** like to keep this file consistent.